tests/plugins/env.rs

```diff
@@ -6,9 +6,9 @@ fn get_env_by_name() {
         cwd: ".",
         plugin: ("nu_plugin_example"),
         r#"
-            $env.FOO = bar
+            $env.FOO = 'bar'
```

Comment:
Here the change probably takes some getting used to.

And if I recall this will diverge from the `FOO=bar` shorthand.

Reply:
Yeah, it would, but it seems over the top to change that too, given that it's a shorthand basically specifically made for that